CIP-0078/CPS-0078.md

```diff
@@ -0,0 +1,41 @@
+---
+CPS: 78
+Title: Cardano Pointer Address Removal
+Status: Open
+Category: Wallets
```

Comment:
@KtorZ I've raised the question before (https://github.com/cardano-foundation/CIPs/pull/372#discussion_r1020125127) about whether there's a category `Core` corresponding to protocol changes.  I am only thinking that if this CPS remained only in the `Wallets` category then it might not be linked with the most important open question (at end of current draft): _Hardfork and not allow them to be used at all for output utxos_?

Reply:
Changed category to `Core` for now.